frontend/public/extend/devconsole/components/pipelines/PipelineVisualizationTask.scss

```
@@ -0,0 +1,118 @@
+$border-color: var(--pf-global--BorderColor--light);
+$gutter: 1.8em;
+
+.odc-pipeline__task {
```

Again, a block and not an element.
```suggestion
.odc-pipeline-vis-task {
```